packages/ui/components/scheduler/components/blocking/BlockedList.tsx

```diff
@@ -0,0 +1,111 @@
+import { useId, useMemo } from "react";
+
+import dayjs from "@calcom/dayjs";
+
+import { useSchedulerStore } from "../../state/store";
+import { BlockedTimeCell } from "./BlockedTimeCell";
+
+type Props = {
+  day: dayjs.Dayjs;
+  containerRef: React.RefObject<HTMLDivElement>;
+};
+
+function roundX(x: number, roundBy: number) {
+  return Math.round(x / roundBy) * roundBy;
+}
+
+type BlockedDayProps = {
+  startHour: number;
+  endHour: number;
+  day: dayjs.Dayjs;
+};
+
+function BlockedBeforeToday({ day, startHour, endHour }: BlockedDayProps) {
+  return (
+    <>
+      {day.isBefore(dayjs(), "day") && (
+        <div
+          key={day.format("YYYY-MM-DD")}
+          className="absolute z-40 w-full"
+          style={{
+            top: `var(--one-minute-height)`,
+            zIndex: 60,
+            height: `calc(${(endHour + 1 - startHour) * 60} * var(--one-minute-height))`, // Add 1 to endHour to include the last hour that we add to display the last vertical line
+          }}>
+          <BlockedTimeCell />
+        </div>
+      )}
+    </>
+  );
+}
```

Comment:
Whenever you are looking at a `day` that is before `today` render a whole empty time cell for the entire grid.